plasma_framework/test/src/framework/utils/ExitPriority.test.js

```diff
@@ -96,6 +98,10 @@ contract('ExitPriority', () => {
             const priority = await this.contract.computePriority(exitableAt, txPos, exitId);
             const parsedExitableAt = await this.contract.parseExitableAt(priority);
             expect(parsedExitableAt).to.be.bignumber.equal(new BN(exitableAt));
+            const parsedExitId = await this.contract.parseExitId(priority);
```

Comment:
Not a fan of just sneaking these extra tests inside the existing ones. It breaks the test description.

I would probably just put this into a new describe, probably `describe('exit priority keeps the exitId and txPos info')` and put a few interested test case there.